src/components/OutputItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { Item } from './SearchOutput/SearchOutput';
+import { useSearchParams } from 'react-router-dom';
+
+type OutputItemProps = {
+  item: Item;
+};
+
+export default function OutputItem({ item }: OutputItemProps) {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  return (
+    <li className="output-item">
+      <span
+        className={`output-item-name${
```

Comment:
it's good to see that you tried to use BEM. but i think, here it should be `output-item__name...`